### PR TITLE
Rename .txt files to match content and add badges.

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
+*******
+Martian
+*******
+
+.. image:: https://travis-ci.org/zopefoundation/martian.svg?branch=master
+        :target: https://travis-ci.org/zopefoundation/martian
+
+.. image:: https://readthedocs.org/projects/martian/badge/?version=latest
+        :target: http://martian.readthedocs.org/en/latest/
```
Comment (mgedmin) [inline README.rst]: Would mildly prefer https://

Note that the link doesn't actually work (neither http:// nor the https:// versions) -- does Martian have any docs uploaded to RTD?

Comment (mgedmin) [reply README.rst]: (Yes, I didn't notice #6 before reviewing.)

**Comment by icemac** [comment]
@mgedmin Thank you for the review. Could you make a release or give me (`icemac`)  the necessary PyPI rights?

**Comment by mgedmin** [comment]
Added `icemac` to PyPI owners for martian.

**Comment by icemac** [comment]
@mgedmin Thank you. I released version 1.0 to PyPI.

**Pull request merged**